tests/features/event.feature

```diff
@@ -269,3 +269,61 @@ Feature: Event content type.
+    And the following Press Contact entity:
+      | Name         | First press contact                                                                      |
+      | Address      | country_code: HU - locality: Szeged - address_line1: Press contact 1 - postal_code: 6700 |
+      | Email        | press1@example.com                                                                       |
+      | Phone number | +32477777777                                                                             |
+    And the following Press Contact entity:
+      | Name         | Second press contact                                                                     |
+      | Address      | country_code: HU - locality: Szeged - address_line1: Press contact 1 - postal_code: 6700 |
+      | Email        | press2@example.com                                                                       |
+      | Phone number | +32477777778                                                                             |
+    And the following General Contact entity:
+      | Name         | A general contact                                                                            |
+      | Address      | country_code: HU - locality: Budapest - address_line1: General contact 1 - postal_code: 1011 |
+      | Email        | general@example.com                                                                          |
+      | Phone number | +32477792933                                                                                 |
+    And the following Event Content entity:
+      | Title   | Event demo page                                              |
+      | Venue   | DIGIT                                                        |
+      | Contact | First press contact, Second press contact, A general contact |
+    When I am visiting the "Event demo page" content
+    Then I should not see the text "DIGIT"
+    And I should not see the text "First press contact"
+    And I should not see the text "Second press contact"
+    And I should not see the text "A general contact"
+    When anonymous users can see events
+    And I reload the page
+    Then I should see the text "DIGIT"
+    And I should see the text "First press contact"
+    And I should see the text "Second press contact"
+    And I should see the text "General contact"
+
+  @preserve_anonymous_permissions
+  Scenario: Unpublished Venues and Contacts are not visible for the visitors.
+    Given anonymous users can see events
+    And the following Default Venue entity:
+      | Name    | Unpublished venue                                                                          |
+      | Address | country_code: BE - locality: Brussels - address_line1: Rue Belliard 28 - postal_code: 1000 |
+      | status  | 0                                                                                          |
```

Comment:
Use published with TRUE/FALSE instead of status and convert it in the class \Drupal\Tests\oe_content\Behat\Content\Venue\DefaultVenueContext::alterVenueFields